.github/workflows/upgrade_test_constraints.yml

```diff
@@ -145,35 +145,22 @@ jobs:
           git remote -v
 
       # START PYTHON DEPENDENCIES
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.9"
-          cache: pip
-          cache-dependency-path: 'pyproject.toml'
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.10"
-          cache: pip
-          cache-dependency-path: 'pyproject.toml'
       - uses: actions/setup-python@v5
         with:
           python-version: "3.11"
           cache: pip
           cache-dependency-path: 'pyproject.toml'
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.12"
-          cache: pip
-          cache-dependency-path: 'pyproject.toml'
 
       - name: Upgrade Python dependencies
         # ADD YOUR CUSTOM DEPENDENCY UPGRADE COMMANDS BELOW
         run: |
-          flags=""
+          set -x
+          pip install -U uv
+          flags="--quiet"
           # Explanation of below commands
-          # python3.8 -m piptools compile - call pip-compile but ensure proper interpreter
+          # uv pip compile --python-version 3.9 - call uv pip compile but ensure proper interpreter
           # --upgrade upgrade to the latest possible version. Without this pip-compile will take a look to output files and reuse versions (so will ad something on when adding dependency.
-          # -o resources/constraints/constraints_py3.8.txt - output file
+          # -o resources/constraints/constraints_py3.9.txt - output file
```

Comment:
this update to 3.9 is a leftover from when we dropped 3.8 in #6826?

Reply:
yes